api/appendix_e.asciidoc

```diff
@@ -561,3 +561,12 @@ Changes from *v3.0.12*:
   * Added missing calls to {clBuildProgram} to the `cl_khr_command_buffer` and `cl_khr_command_buffer_mutable_dispatch` sample code.
   * Fixed a copy-paste error in the extensions quick reference appendix.
   * Fixed typos and improved formatting consistency in the extensions spec.
+
+Changes from *v3.0.13*:
+
+  * Corrected the precision for `cross` and `dot` to be based on `HALF_EPSILON` in `cl_khr_fp16`.
+  * Added a context query for command-buffers to `cl_khr_command_buffer`.
+  * Updated the semaphore wait and signal rules in `cl_khr_semaphore`.
```

Comment:
I'm wondering whether we shouldn't say something a bit more descriptive here. Here's a straw-man:

> Changed the behavior of wait and signal enqueue commands in `cl_khr_semaphore`; they may now block under some circumstances (see KhronosGroup/OpenCL-Docs#883 for background).

I _think_ this would be the first occurrence of a Github issue link in the spec but IMO that'd be a good habit to get into. It'd make looking at historical discussions a bit easier than having to search through the git history (which admittedly isn't typically that hard).

Reply:
Interesting idea, I added GitHub PRs for each of the items in the change log.  I didn't actually make them links, though I suppose we could do that too.  Let me know what you think.

Note, I added a bit more description to the semaphore item too, but I didn't say "blocking" because I think it's too easy to misunderstand in isolation.

Reply:
I think links would be better for people for whom it wouldn't be obvious that KhronosGroup/OpenCL-Docs is the project the number are valid for. Also makes it easier to refer to PRs and Issues. Regarding the semaphore change I was more after something that would communicate the gist of the change in one line (which is hard to do! :)). We're talking polishing here.

Reply:
I added https://github.com/KhronosGroup/OpenCL-Docs/issues/911 so we remember to add the PR and issue links.  Thanks!